### DISCUSSION TITLE
Welcome to mcjoin Discussions!

### DISCUSSION THREAD

**Discussion by troglobit** [troglobit/mcjoin#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by neirbowj** [comment]
Hello @troglobit and other mcjoin users. 👋 

I am the maintainer of the mcjoin port on FreeBSD and have enjoyed using it to test multicast network protocols in my employer's networking lab.

**Reply by troglobit** [reply]
Cool, nice to hear back from you! :sunglasses::wave:
